packages/framework/resources/views/layouts/head.blade.php

```diff
@@ -16,4 +16,7 @@
     {{-- Check the local storage for theme preference to avoid FOUC --}}
     <meta id="meta-color-scheme" name="color-scheme" content="{{ config('hyde.default_color_scheme', 'light') }}">
     <script>if (localStorage.getItem('color-theme') === 'dark' || (!('color-theme' in localStorage) && window.matchMedia('(prefers-color-scheme: dark)').matches)) { document.documentElement.classList.add('dark'); document.getElementById('meta-color-scheme').setAttribute('content', 'dark');} else { document.documentElement.classList.remove('dark') } </script>
-@endif
\ No newline at end of file
+@endif
+
+{{-- If the user has defined any custom head tags, render them here --}}
+{!! \Hyde\Support\Includes::blade('meta', config('hyde.hooks.head')) !!}
```

Comment:
Optimally, this would support both Blade and Html files, but we don't have an HTML include (yet), and then the coalesce chain would become so long that we'd want to move it to a helper method. So maybe for later.